src/components/modal.js

```diff
@@ -0,0 +1,66 @@
+// outsource dependencies
+import React from 'react';
+import PropTypes from 'prop-types';
+import { BsExclamationCircle } from 'react-icons/all';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Row, Col } from 'reactstrap';
+
+const CustomModal = (props) => {
+  const {
+    title,
+    isOpen,
+    onCancel,
+    onSuccess,
+    successBtnLabel,
+    cancelBtnLabel,
+    ...rest
+  } = props;
+
+  const closeBtn = <button
+    onClick={onCancel}
+    style={{ fontSize: '2.5em' }}
+    className="btn close pb-2 text-light">×
+  </button>;
+
+  return <div>
+    <Modal isOpen={isOpen} toggle={onCancel} {...rest} centered={true}>
```

Comment:
```suggestion
    <Modal isOpen={isOpen} toggle={onCancel} {...rest} centered>
```

It should be centered by default